src/Consolonia.Themes/Templates/Controls/SelectableTextBlock.axaml

```diff
@@ -0,0 +1,11 @@
+<ResourceDictionary xmlns="https://github.com/avaloniaui"
+                    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+                    xmlns:helpers="clr-namespace:Consolonia.Themes.Templates.Controls.Helpers">
+    <ControlTheme x:Key="{x:Type SelectableTextBlock}"
+                  TargetType="SelectableTextBlock">
+        <Setter Property="SelectionBrush"
+                Value="{DynamicResource ThemeActionBackgroundBrush}" />
+        <Setter Property="helpers:SelectTextWithPointerUpExtension.SelectOnMouseLeftUp"
```

Comment:
next level magic. 

I haven't ever created a mixin extension like this.  Xaml is so freaking powerful as a model.

Reply:
True. Its supposed to be simpler with "behaviours", they manage lifetime etc, but Avalonia does not have them builtin, i dont want to include that assembly to core level packages of consolonia.